core/function.rs

```diff
@@ -104,6 +104,22 @@ pub enum Func {
     Json(JsonFunc),
 }
 
+impl Func {
+    pub fn to_string(&self) -> String {
+        match self {
+            Func::Agg(agg_func) => agg_func.to_string().to_string(),
```

Comment:
double to_string?

Reply:
yeah `AggFunc` has a separate non-trait method `to_string()` that returns a `&str`, should change that at some point 😂

Reply:
ah okay